src/Microsoft.Diagnostics.Runtime/src/Builders/RuntimeBuilder.cs

```diff
@@ -1057,6 +1050,8 @@ private void CreateFieldsForMethodTableWorker(ClrType type, out ImmutableArray<C
             statics = staticsBuilder.MoveOrCopyToImmutable();
         }
 
+        public ClrDataModule? GetClrDataModule(ClrModule module) => _sos.GetClrDataModule(module.Address);
```

Comment:
I am not sure whether we should expose this through ClrModule or not.  For example, ClrThread doesn't expose `ClrStackWalk` which is similar to this.  I would suggest using ClrDataModule/UntitledModuleData to fill in the properties of ClrModule more accurately if it exists, but not directly expose it through ClrModule's API surface.